src/policy/policy.cpp

```diff
@@ -51,7 +51,8 @@ CAmount GetDustThreshold(const CTxOut& txout, const CFeeRate& dustRelayFeeIn)
 
 bool IsDust(const CTxOut& txout, const CFeeRate& dustRelayFeeIn)
 {
-    return (txout.nValue < GetDustThreshold(txout, dustRelayFeeIn));
+    return (txout.nValue < GetDustThreshold(txout, dustRelayFeeIn) 
+    && !(txout.nValue == 1 and GetColorIdFromScript(txout.scriptPubKey).type == TokenTypes::NFT));
```

Comment:
Since a small number of tokens can be traded outside of the NFT, let's keep the tokens out of the Dust. If so, is there a problem?

Reply:
no, it think I can check the script instead and not check for dust in case it is token.